README.md

```diff
@@ -1 +1,93 @@
 # Yugabyte Kubernetes Operator
+
```

Comment:
As synced up offline, we need to capture more information around pre-req on operator-sdk and also instructions on how to build and deploy and replace the image_tag etc needs to be documented.

Reply:
Added `CONTRIBUTING.md` to the PR